src/layouts/BaseLayout.astro

```diff
@@ -1,27 +1,31 @@
 ---
-import Footer from "@/src/components/Footer/index.astro";
+import Footer from "@components/Footer/index.astro";
 import Nav from "@components/Nav/index.astro";
 import Settings from "@components/Settings/index.astro";
 import RootPageHeader from "@components/RootPageHeader/index.astro";
 import ItemPageHeader from "@components/ItemPageHeader/index.astro";
 import { getCurrentLocale, useTranslations } from "@i18n/utils";
 import "@styles/base.scss";
 
-interface BaseLayoutProps {
+interface Props {
   title: string;
   className?: string;
-  variant: "root" | "item";
+  variant?: "root" | "item";
+  localizeTitles?: boolean;
```

Comment:
I guess I wonder if this is all to avoid [that console warning](https://github.com/bocoup/p5.js-website/blob/main/src/i18n/utils.ts#L147)? I almost feel like we should remove that anyway since it is so noisy during build and the fallback to the key is an important feature of the function.